pyvisa-py/sessions.py

```diff
@@ -262,14 +262,10 @@ def gpib_command(self, command_byte):
 
         :param command_byte: command byte to send
         :type command_byte: int, must be [0 255]
-        :return: return value of the library call
-        :rtype: :class:`pyvisa.constants.StatusCode`
+        :return: Number of written bytes, return value of the library call.
+        :rtype: int, :class:`pyvisa.constants.StatusCode`
         """
-        try:
-            return self.sessions[session].gpib_command(command_byte)
-
-        except KeyError:
-            return constants.StatusCode.error_invalid_object
+        return StatusCode.error_nonsupported_operation
```

Comment:
This looks incorrect to me we need to return `0, StatusCode.error_nonsupported_operation` as per the docstring which is correct.

Reply:
Correct, I missed that one. Will update the PR.